CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing to TRIC-DT repo and Knowledge Commons
+
+Welcome to the TRIC-DT GitHub repository!
+
+The point of this file is to make it easy for you to get involved in our activities and contribute to these materials. If you have any questions that aren't covered here please let us know via an issue, pull request, or contact us! 
+
+
+**Contents**
+- [Why contribute](#why-contribute)
+- [Why GitHub](#why-github)
+- [Issues](#issues)
+- [Pull request](#pull-requests-pr)
+- [Participate in our activities](#particpate-in-our-activities)
+- [Code of conduct](#code-of-conduct)
+- [Contact us](#contact-us)
+- [Acknowledgements](#acknowledgements)
+
+## Why contribute
+
+**A key mission of TRIC-DT is to [democratise access to open, reproducible and trustworthy ecosystems of digital twins](https://www.turing.ac.uk/research/research-projects/tric-dt).**
+
+**This is best acheived through broad and open collaboration in the materials we create and share.**
+
+**We invite you to contribute to these materials and our activites in any way you can, to make sure your perspecties and expertise are included, supporting the development of robust, accessible and equitable outputs.**
+
+## Why GitHub
+
+These materials are being developed in GitHub to ensure atribution and version control. Take a look at the [Github for Collaborative Documentation](https://cassgvp.github.io/github-for-collaborative-documentation/) for a quick overview on why we recomend this processs, and [The Turing Way](https://the-turing-way.netlify.app/index.html) for a more in-depth discussion.
+
+### Github account
+Before you can contribute formally (i.e. add to these materials directly), you'll need a [GitHub account (free)](https://github.com). 
+
+### GitHub contribution processes
+
+You can contribute by [getting in touch with us](#contact-us) and we will record your engagement using  [all-contributors](https://allcontributors.org).
```

Comment:
Are these really connected? We don't record _all_ engagement through all-contributors, or is this the point of using all-contributors, which I might have missed?